metadata_annotation_farland.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                                    cell_line_source='Cancerrxgene')
</pre><p>wow, it's really called "cancerrxgene"? yeesh</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>